gemfire-shared/src/main/java/com/gemstone/gemfire/internal/shared/Version.java

```diff
@@ -227,11 +229,17 @@
   public static final Version GFXD_20 = new Version("GFXD", "2.0",
       (byte)2, (byte)0, (byte)0, (byte)0, GFXD_20_ORDINAL, GFE_75);
 
+  private static final byte GFE_80_ORDINAL = GFXD_14_ORDINAL;
+  public static final Version GFE_80 = new Version("GFE", "8.0", (byte)8,
+      (byte)0, (byte)0, (byte)0, GFE_80_ORDINAL);
+
```

Comment:
Seems safer to not overwrite GFXD_14 version. So if we can have a constructor argument to Version constructor to not overwrite VALUES global then it will be better. So then GFE_80 will be a temporary Version object which is only inside GFXD_155. Something like:

```java
   public static final Version GFE_80 = new Version("GFE", "8.0", (byte)8,
      (byte)0, (byte)0, (byte)0, GFE_80_ORDINAL, false /* overwrite */);
  ...
  /** Creates a new instance of <code>Version</code> */
  private Version(String product, String name, byte major, byte minor,
      byte release, byte patch, byte ordinal) {
    this(product, name, major, minor, release, patch, ordinal, true /* overwrite */);
  }

  private Version(String product, String name, byte major, byte minor,
      byte release, byte patch, byte ordinal, boolean overwrite) {
    ...
    if (overwrite && ordinal != TOKEN_ORDINAL) {
      VALUES[this.ordinal] = this;
    }
  }
```